config/webpacker.yml

```diff
@@ -33,6 +33,9 @@ development:
   <<: *default
   compile: true
 
+  # Verifies that versions and hashed value of the package contents in the project's package.json
+  check_yarn_integrity: true
```

Comment:
Are we using yarn now?

Reply:
Yes, we can use yarn now that we are no longer using Manifesto. I have changed all the relevant files, but need to still update the figgy ansible playbook.

Reply:
Actually, if I understand correctly, it looks like yarn is already being installed... https://github.com/pulibrary/princeton_ansible/blob/master/playbooks/figgy.yml#L43